### PR TITLE
[FEATURE] #97667 - Add keyboard support for Multiselect

### PR DESCRIPTION
**Pull request by brotkrueml**
As this manual is for editors, the different types are explained by screenshots where the keyboard commands can be used.

Additionally, a TOC has been added.

Resolves: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/412
Releases: main, 12.4

### CODE REVIEW

File: Documentation/HelpInside/KeyboardCommands/Index.rst [hunk 1]
```diff
@@ -77,3 +83,50 @@ Opening the backend search modal
...
+    :class: with-shadow
+
+    A group element
+
+Selecting and deselecting options with the keyboard:
+
+*   :kbd:`enter` = Add options, either from right to left or left to right
+*   :kbd:`delete` or :kbd:`backspace` = Remove an option for Windows and Mac users
+*   :kbd:`alt` + :kbd:`arrow up` = Move the option one up
+*   :kbd:`alt` + :kbd:`arrow up` = Move the option one up
```
Comment (garvinhicking) [inline Documentation/HelpInside/KeyboardCommands/Index.rst:122]: That's a duplicate?

File: Documentation/HelpInside/KeyboardCommands/Index.rst [hunk 2]
```diff
@@ -77,3 +83,50 @@ Opening the backend search modal
...
+*   :kbd:`alt` + :kbd:`arrow down` = Move the option one down
```
Comment (garvinhicking) [inline Documentation/HelpInside/KeyboardCommands/Index.rst:123]: Would have to look up if this is done via cmd on mac?

**Review by garvinhicking — COMMENTED** [review]
(Not fully reviewed yet, out of time)

File: Documentation/HelpInside/KeyboardCommands/Index.rst (hunk 1 above)
Comment (brotkrueml) [reply Documentation/HelpInside/KeyboardCommands/Index.rst:122]: oops

File: Documentation/HelpInside/KeyboardCommands/Index.rst (hunk 2 above)
Comment (brotkrueml) [reply Documentation/HelpInside/KeyboardCommands/Index.rst:123]: cmd is equivalent to ctrl -> so it is safe

File: Documentation/HelpInside/KeyboardCommands/Index.rst (hunk 1 above)
Comment (brotkrueml) [reply Documentation/HelpInside/KeyboardCommands/Index.rst:122]: done

**Review by garvinhicking — APPROVED** [review]

**Review by linawolf — APPROVED** [review]

**Pull request merged**